node/impl/full/eth.go

```diff
@@ -1628,13 +1646,15 @@ func (a *EthModule) EthCall(ctx context.Context, tx ethtypes.EthCall, blkParam e
 	}
 
 	if msg.To == builtintypes.EthereumAddressManagerActorAddr {
-		// As far as I can tell, the Eth API always returns empty on contract deployment
 		return ethtypes.EthBytes{}, nil
 	} else if len(invokeResult.MsgRct.Return) > 0 {
-		return cbg.ReadByteArray(bytes.NewReader(invokeResult.MsgRct.Return), uint64(len(invokeResult.MsgRct.Return)))
+		result, err = cbg.ReadByteArray(bytes.NewReader(invokeResult.MsgRct.Return), uint64(len(invokeResult.MsgRct.Return)))
```

Comment:
What is this change doing ? I don't see us returning an `ExecutionRevertedError` anywhere.

Reply:
It was added just to add more context in the error:
 `return nil, xerrors.Errorf("failed to read msg recipient: %w", err)`
 
`ExecutionRevertedError` will be returned by the `applyMessage` directly as it was already returning the `execution reverted` error.